core-api/src/main/java/com/sequenceiq/cloudbreak/type/OpenStackCredentialConstants.java

```diff
@@ -0,0 +1,23 @@
+package com.sequenceiq.cloudbreak.type;
+
+/**
+ * Created by gpapp on 6/14/18.
+ */
+public class OpenStackCredentialConstants {
```

Comment:
Instead of an enum?

Reply:
I do like enums. Still, I think string constants do lead the eye during reading better than enums. What do you think?